### DISCUSSION TITLE
Should we add a message for users to update keploy to the latest version?

### DISCUSSION THREAD

**Discussion by SkySingh04** [keploy/keploy#1617] [Polls]
References:
Issue: #1485 
Pull Request: #1486

Here we can consider the update message to be of the following format:
![image](https://github.com/keploy/keploy/assets/114267538/852b4596-7b01-47b6-bb3c-5af8e536f628)

## Key points to note:
- The message will not show up on dev and latest versions
- There will be a **delay** that will happen(~2-3s) due to an Http request being made everytime keploy is run to fetch the latest version.
- Potential solutions for the delay can be:

1. The update message is shown at the end of execution of the command
2. The update message is shown only once a day.

Looking forward to an engaging discussion with the community!

**Comment by re-Tick** [comment]
The feature will be helpful for the users. To avoid the delay, it can be like the ohmyzsh where an update prompt is asked when the new terminal is initiated.

**Reply by SkySingh04** [reply]
That sounds like a perfect solution. Will implement it right away
